src/d/actor/d_a_obj_hat.cpp

```diff
@@ -4,91 +4,238 @@
 //
 
 #include "d/actor/d_a_obj_hat.h"
+#include "d/actor/d_a_npc_roten.h"
+#include "d/d_a_obj.h"
+#include "d/d_com_inf_game.h"
+#include "d/d_kankyo.h"
 #include "d/d_procname.h"
+#include "d/res/res_ro.h"
+
+// const char daObjHat_c::m_arcname[] = "ObjHat";
```

Comment:
```suggestion
// const char daObjHat_c::m_arcname[] = "ObjHat";
```
Why is this commented out?